hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/security/AWSV4AuthValidator.java

```diff
@@ -39,6 +40,11 @@ final class AWSV4AuthValidator {
   private static final Logger LOG =
       LoggerFactory.getLogger(AWSV4AuthValidator.class);
   private static final String HMAC_SHA256_ALGORITHM = "HmacSHA256";
+  private static final Pattern STR_TO_SIGN_PATTERN =
+          Pattern.compile("^AWS4[A-Z\\-\\d]+\n" +
```

Comment:
```suggestion
          Pattern.compile("^AWS4-HMAC-SHA256\n" +
```
We could add `AWS4-HMAC-SHA256` towards the beginning of the string to adhere by the AWS V4 signature format.